vignettes/ebola.Rmd

```diff
@@ -0,0 +1,539 @@
+
+head(obs_prep)
+```
+
+Now we are ready to fit the latent individual model.
+
+### Intercept-only model
+
+We start by fitting a single lognormal distribution to the data. 
+This model assumes that a single distribution describes all delays in the data, regardless of the case's location, sex, or any other covariates.
+To do this, we set `formula = brms::bf(mu ~ 1, sigma ~ 1)` to place an model with only and intercept parameter (i.e. `~ 1` in R formula syntax) on the `mu` and `sigma` parameters of the lognormal distribution.
+This distribution is specified using `family = brms::lognormal()`.
+
+```{r}
+fit <- epidist(
+  data = obs_prep,
+  formula = brms::bf(mu ~ 1, sigma ~ 1),
+  family = brms::lognormal(),
+  algorithm = "sampling",
+  refresh = 0,
+  silent = 2,
+  seed = 1
+)
+```
+
+The `fit` object is a [`brmsfit`](https://paulbuerkner.com/brms/reference/brmsfit-class.html) object, and has the associated range of methods.
+See `methods(class = "brmsfit")` for more details.
+For example, we may use `summary` to view information about the fitted model, including posterior estimates for the regression coefficients:
+
+```{r}
+summary(fit)
+```
+
+### Sex-stratified model
+
+To fit a model which varies the parameters of the fitted lognormal distribution, `mu` and `sigma`, by sex we alter the `formula` specification to include fixed effects for sex `~ 1 + sex` as follows:
+
+```{r}
+fit_sex <- epidist(
+  data = obs_prep,
+  formula = brms::bf(mu ~ 1 + sex, sigma ~ 1 + sex),
+  family = brms::lognormal(),
+  algorithm = "sampling",
+  refresh = 0,
+  silent = 2,
+  seed = 1
+)
+```
+
+A summary of the model shows that males tend to have longer delays (the posterior mean of `sexM` is `r round(summary(fit_sex)$fixed[3, "Estimate"], 2)`) and greater delay variation (the posterior mean of `sigma_sexM` is `r round(summary(fit_sex)$fixed[4, "Estimate"], 2)`).
+For the `sexM` effect, the 95% credible interval is greater than zero, whereas for the `sigma_sexM` effect the 95% credible interval includes zero.
+It is important to note that the estimates represent an average of the observed data, and individual delays between men and women vary significantly.
+
+```{r}
+summary(fit_sex)
+```
+
+### Sex-district stratified model
+
+Finally, we will fit a model which also varies by district.
+To do this, we will use district level random effects, assumed to be drawn from a shared normal distribution, within the model for both the `mu` and `sigma` parameters.
+These random effects are specified by including `(1 | district)` in the formulas:
+
+```{r}
+fit_sex_district <- epidist(
+  data = obs_prep,
+  formula = brms::bf(
+    mu ~ 1 + sex + (1 | district),
+    sigma ~ 1 + sex + (1 | district)
+  ),
+  family = brms::lognormal(),
+  algorithm = "sampling",
+  refresh = 0,
+  silent = 2,
+  seed = 1
+)
+```
+
+For this model, along with looking at the `summary`, we may also use the `brms::ranef` function to look at the estimates of the random effects:
```

Comment:
In f2f you said you preferred just doing `package::function` in the text and not the code but here I see it everywhere. Note that using `brms::ranef()` will auto link in `pkgdown` but I think the current approach will not?

Reply:
Added to https://github.com/epinowcast/epidist/issues/290